pkg/occlient/occlient_test.go

```diff
@@ -448,3 +449,62 @@ func Test_parseImageName(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateServiceInstance(t *testing.T) {
+	type args struct {
+		componentName   string
+		componentType   string
+		applicationName string
+		labels          map[string]string
+	}
+
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "Create service instance",
+			args: args{
+				componentName: "jenkins",
+				componentType: "jenkins",
+				labels: map[string]string{
+					"name":      "mongodb",
+					"namespace": "blog",
+				},
+			},
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			fkclient, fkclientset := FakeNew()
+
+			//_, err := fkclient.CreatePVC(tt.name, tt.size, tt.labels)
+			err := fkclient.CreateServiceInstance(tt.args.componentName, tt.args.componentType, tt.args.labels)
+			// Checks for error in positive cases
+			if tt.wantErr == false && (err != nil) {
+				t.Errorf(" client.CreateServiceInstance(componentName,componentType, labels) unexpected error %v, wantErr %v", err, tt.wantErr)
+			}
+
+			// Check for validating actions performed
+			if len(fkclientset.ServiceCatalogClientSet.Actions()) != 1 && tt.wantErr == false {
+				t.Errorf("expected 1 action in CreateServiceInstace got: %v", fkclientset.ServiceCatalogClientSet.Actions())
+			}
+
```

Comment:
The checks below would be necessary only if `err` is `nil`. You could add that check here?